Disable plugins for which no dependency is checked

Disable resthttpdevice plugin broken with recent boost versions
ref https://github.com/openscenegraph/OpenSceneGraph/issues/921

--- src/osgPlugins/CMakeLists.txt~	2026-08-04 14:53:45.158560459 +0200
+++ src/osgPlugins/CMakeLists.txt	2026-08-04 14:54:13.237119656 +0200
@@ -103,13 +103,10 @@
     ADD_PLUGIN_DIRECTORY(jpeg)
 ENDIF()
 IF(JASPER_FOUND)
-    ADD_PLUGIN_DIRECTORY(jp2)
 ENDIF()
 IF(OPENEXR_FOUND AND ZLIB_FOUND AND OSG_CPP_EXCEPTIONS_AVAILABLE)
-    ADD_PLUGIN_DIRECTORY(exr)
 ENDIF()
 IF(GIFLIB_FOUND)
-    ADD_PLUGIN_DIRECTORY(gif)
 ENDIF()
 IF(PNG_FOUND AND OSG_CPP_EXCEPTIONS_AVAILABLE)
     ADD_PLUGIN_DIRECTORY(png)
@@ -118,8 +115,6 @@
     ADD_PLUGIN_DIRECTORY(tiff)
 ENDIF()
 IF(GDAL_FOUND)
-    ADD_PLUGIN_DIRECTORY(gdal)
-    ADD_PLUGIN_DIRECTORY(ogr)
 ENDIF()
 IF(GTA_FOUND AND OSG_CPP_EXCEPTIONS_AVAILABLE)
     ADD_PLUGIN_DIRECTORY(gta)
@@ -210,11 +205,9 @@
 ENDIF()
 
 IF(FFMPEG_FOUND AND OSG_CPP_EXCEPTIONS_AVAILABLE)
-    ADD_PLUGIN_DIRECTORY(ffmpeg)
 ENDIF()
 
 IF(GSTREAMER_FOUND AND GLIB_FOUND)
-    ADD_PLUGIN_DIRECTORY(gstreamer)
 ENDIF()
 
 IF(DIRECTSHOW_FOUND)
@@ -255,15 +248,12 @@
 ENDIF()
 
 IF(RSVG_FOUND)
-    ADD_PLUGIN_DIRECTORY(svg)
 ENDIF()
 
 IF(POPPLER_FOUND)
-    ADD_PLUGIN_DIRECTORY(pdf)
 ENDIF()
 
 IF(LIBVNCSERVER_FOUND)
-    ADD_PLUGIN_DIRECTORY(vnc)
 ENDIF()
 
 ADD_PLUGIN_DIRECTORY(pvr)
@@ -300,11 +290,9 @@
 # Device integration plugins
 #
 IF   (SDL_FOUND)
-    ADD_PLUGIN_DIRECTORY(sdl)
 ENDIF(SDL_FOUND)
 
 IF(ASIO_FOUND)
-    ADD_PLUGIN_DIRECTORY(RestHttpDevice)
 ENDIF(ASIO_FOUND)
 
 
